studio/src/components/feature-flag-details.tsx

```diff
@@ -64,6 +66,24 @@ const FeatureFlagOverview = ({
         }
       />
     );
+  } else if (!isEnabled) {
+    content = (
+      <EmptyState
+        icon={<ExclamationTriangleIcon className="text-red-500" />}
+        title="Feature flag is not used for composition."
+        description={
+          <>
+            Feature flag is disabled. To enable the feature flag, use the below
```

Comment:
Is there meant to be the ability for enabling the feature flag from the UI as well?

Reply:
No

Reply:
Nothing currently